core/src/main/java/io/kestra/plugin/core/http/Request.java

```diff
@@ -38,7 +38,7 @@
 @Plugin(
     examples = {
         @Example(
-            title = "Execute a Kestra flow via an HTTP POST request authenticated with basic auth. To pass a `user` input to the API call, we use the `formData` property. When using form data, make sure to set the `contentType` property to `multipart/form-data`.",
+            title = "Execute a Kestra flow via a HTTP POST request authenticated with basic auth. To pass a `user` input to the API call, we use the `formData` property. When using form data, make sure to set the `contentType` property to `multipart/form-data`.",
```

Comment:
```suggestion
            title = "Execute a Kestra flow via an HTTP POST request authenticated with basic auth. To pass a `user` input to the API call, we use the `formData` property. When using form data, make sure to set the `contentType` property to `multipart/form-data`.",
```
it's really an HTTP, not a :) I can recommend Grammarly to catch such spelling issues